com.atlauncher.ATLauncher.yml

```
@@ -0,0 +1,63 @@
+# Credits to github/nickavem for the script, Doomsdayrs for edits
+
+app-id: com.atlauncher.ATLauncher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.openjdk8
+command: atlauncher.sh
+build-options:
+    env:
+        PATH: /app/jre/bin:/usr/bin
+finish-args:
+    - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+    - --socket=x11
+    - --share=ipc
+    - --device=all
+    - --socket=pulseaudio
+    - --share=network
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+          - /usr/lib/sdk/openjdk8/install.sh
+
+    - name: atlauncher
+      buildsystem: simple
+      build-commands:
+          - ls -a
+          - install -D ATLauncher.jar /app/bin/ATLauncher.jar
+      sources:
+          - type: file
+            dest-filename: ATLauncher.jar
+            url: https://github.com/ATLauncher/ATLauncher/releases/download/v3.4.10.15/ATLauncher-3.4.10.15.jar
+            tag: v3.4.10.15
+            sha256: 4082ecc4e9dfc824a64f201034640cb65a66a65ef6518cd2389d98b7744f4a0f   
+            x-checker-data:
+             type: git
+             url: https://github.com/ATLauncher/ATLauncher.git
+             tag-pattern: ^([\d.]+)$
+        
+
+    - name: atlauncher.sh
+      buildsystem: simple
+      build-commands:
+          - install -D atlauncher.sh /app/bin/atlauncher.sh
+      sources:
+          - type: file
+            path: atlauncher.sh
+
+    - name: atlauncher-extra
+      buildsystem: simple
+      build-commands:
+          - install -D com.atlauncher.ATLauncher.desktop /app/share/applications/com.atlauncher.ATLauncher.desktop
+          - install -D com.atlauncher.ATLauncher.metainfo.xml /app/share/metainfo/com.atlauncher.ATLauncher.appdata.xml
+          - install -D icon.svg /app/share/icons/hicolor/scalable/apps/com.atlauncher.ATLauncher.svg
```

```suggestion
          - install -D images/icon.svg /app/share/icons/hicolor/scalable/apps/com.atlauncher.ATLauncher.svg
```

Given the path below.

---

because things changed between the time I made the comment and the time your replied to it. It's no longer valid it seems.